test/test.js

```diff
@@ -89,21 +94,6 @@ describe('/POST /v1/users',() => {
 });
 
 
-
-describe('/POST /v1/users',() => {
-    it.skip("should find users even without any parameter", (done) => {
```

Comment:
Fixing this with #14  - can we leve it as is ? no strong opinions if you still want it out, just LMK